iron-list.html

```diff
@@ -1245,6 +1262,35 @@
     _multiSelectionChanged: function(multiSelection) {
       this.clearSelection();
       this.$.selector.multi = multiSelection;
+    },
+
+    _scrollerChanged: function(scroller, oldScroller) {
+      if (scroller === 'window') {
+        this.scroller = window;
+      } else if (scroller) {
+        if (!this._scrollListener) {
+          this._scrollListener = IOS_TOUCH_SCROLLING ?
+            function() {
+              requestAnimationFrame(this._scrollHandler.bind(this));
+            }.bind(this) : this._scrollHandler.bind(this);
+        }

```

Comment:
This code was moved from `ready` to here, I suppose because `_scrollListener` needs to be defined before `ready` since `scroller` may be set via an attribute; in that case, the old `ready` code can just be changed to `created`.  Probably clearer to do it in `created` than the one-shot code here in the changed handler.